jsapp/js/components/modalForms/translationSettings.es6

```diff
@@ -269,17 +270,34 @@ export class TranslationSettings extends React.Component {
                 </a>
               </p>
             </bem.FormView__cell>
-            :
-            <bem.FormView__cell m='label'>
-              {t('Current languages')}
+            <bem.FormView__cell m='translation'>
+              <p><strong>{t('Please name your default language before adding languages and translations.')}</strong></p>
             </bem.FormView__cell>
-          }
+          </React.Fragment>
+          <bem.FormView__cell m='add-language-form'>
```

Comment:
I think it would look better with `m='update-language-form'` :)

Reply:
OK, then I've also made the one in the translation table an "update language form" so it fits